devops/create_https_wss_endpoints_aws/master.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain_name": "master.hmny.io",
```

Comment:
what's this? do we have a master.hmny.io? Or it is just an example.

Reply:
it is for testing